### DISCUSSION TITLE
Version for windows XP

### DISCUSSION THREAD

**Discussion by DonDakin** [asb2m10/dexed#330] [General]
Hello,

I have an older windows XP machine and I wonder if there is a compiled binary for XP out there that I can get a hold of.

Thanks

Don....

**Comment by asb2m10** [comment]
I think Dexed 0.8.0 was XP compatible; check here http://www.le-son666.com/software/dexed/index.html

**Comment by DonDakin** [comment]
Salut Pascal,

Thanks for the direction. I was able to install this version as a VST plugin on my old Sonar 4 system on XP. 
I have a question: 
I see the predefined patches that I can select but is it possible to change these patches the same way we can change the carts in the latest version ?

Thanks again

Don,,,,,,
